### ISSUE TITLE
Writing json - Force parameter as type

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/GiDInterface#99]
Some system must be defined to make sure that a variable is written in the json as it's desired type.

A developer should be able to write:

```json
"name": "3",
"value": 3,
"other": 3.0
```
where the 3 is considered first as a string, then as an integer, and as a double.

**Comment by pooyan-dadvand** [comment]
As long as I know Json already distinguish them in this way.

**Comment by jginternational** [comment]
Yes @pooyan-dadvand , but tcl is proudly a non-typed language, so it's always difficult to know if that 3 is an integer, a double or a string

**Comment by pooyan-dadvand** [comment]
Now I see, where we should add the types, in XMLs? We should try to make it backward compatible as an optional attribute.
